Singer.API/ClientApp/src/app/modules/admin/components/careusers/overview/overview.component.ts

```diff
@@ -105,12 +105,12 @@ export class OverviewComponent implements OnInit, AfterViewInit {
             debounceTime(400),
             distinctUntilChanged(),
             tap(() => {
-               this.paginator.pageIndex = 1;
```

Comment:
Back to page index 0? I thought we agreed to use a 1-base page index (not that I disaprove)?

Reply:
I introduced 0-based a couple of tickets ago, yet forgot to change these sortchange events.